frontend/README.md

```diff
@@ -0,0 +1,14 @@
+# URL-S Frontend
+
+Instructions for configuring and starting the URL-S frontend app, Vue.js, and its dependencies.
+
+It is recommended that you install and start the Backend applications before using the Frontend. See the README in the Backend folder.
```

Comment:
recommended should be changed to necessary as it is a dependent step.

Reply:
I added text in Commit d16d4d9aecf196fa52aaebfe84b56646563df241

"Before doing the the frontend install and start, the Backend application must be installed and started. Follow instructions in..."